tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/OpenPaasContactsConsumer.java

```diff
@@ -42,9 +42,13 @@
 public class OpenPaasContactsConsumer implements Startable, Closeable {
     private static final Logger LOGGER = LoggerFactory.getLogger(OpenPaasContactsConsumer.class);
     private static final boolean REQUEUE_ON_NACK = true;
-    public static final String EXCHANGE_NAME = "contacts:contact:add";
-    public static final String QUEUE_NAME = "openpaas-contacts-queue";
-    public static final String DEAD_LETTER = QUEUE_NAME + "-dead-letter";
+    public static final String EXCHANGE_NAME_ADD = "contacts:contact:add";
+    public static final String EXCHANGE_NAME_UPDATE = "contacts:contact:update";
+    public static final String EXCHANGE_NAME_DELETE = "contacts:contact:delete";
+    public static final String QUEUE_NAME_ADD = "openpaas-contacts-queue-add";
+    public static final String QUEUE_NAME_UPDATE = "openpaas-contacts-queue-update";
+    public static final String QUEUE_NAME_DELETE = "openpaas-contacts-queue-delete";
+    public static final String DEAD_LETTER = "openpaas-contacts-queue-dead-letter";
```

Comment:
With 3 queue but only one dead letter how are we going to sort things out easily from deadletter and reprocess from correct location?